code/datums/mind.dm

```diff
@@ -53,6 +53,9 @@
 	/// Tracks if this mind has been a rev or not
 	var/has_been_rev = FALSE
 
+	/// Tracks when they fail the probability of head antagonist, so they don't get multiple rolls.
+	var/failed_head_antag_roll = FALSE
```

Comment:
Might be better to make this a (lazy) list on the SSjobs datum instead. And clean it after starting the game

Reply:
Done!